src/control/server/server.go

```diff
@@ -69,6 +70,12 @@ func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 	// Backup active config.
 	saveActiveConfig(log, cfg)
 
+	if cfg.HelperLogFile != "" {
+		if err := os.Setenv(pbin.DaosAdminLogFileEnvVar, cfg.HelperLogFile); err != nil {
+			return errors.Wrap(err, "unable to configure privileged helper logging")
```

Comment:
What's the resolution if this error occurs?  Can you put something in the error message that helps the admin understand what to do differently?

Reply:
That's one of those "can't happen" errors... I have no idea what would cause os.Setenv() to fail, but the function returns an error so I'm handling it. I'm assuming that if it fails there is no practical resolution other than "fix your system because it's really screwed up".

Reply:
FWIW: Here's the implementation that's called on Linux: https://golang.org/src/syscall/env_unix.go?s=1939:1975#L83

So basically there are some potential failures which are due to programmer error. If those somehow get out to be seen by an admin in the wild then we've really messed up.

Reply:
Let's go with that.